src/main/java/edu/hawaii/its/groupings/configuration/OotbSecurityConfig.java

```diff
@@ -0,0 +1,80 @@
+package edu.hawaii.its.groupings.configuration;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+import static org.springframework.security.web.util.matcher.AntPathRequestMatcher.antMatcher;
+
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
+import org.springframework.ws.config.annotation.DelegatingWsConfiguration;
+
+import edu.hawaii.its.groupings.access.DelegatingAuthenticationFailureHandler;
+
+@EnableWebSecurity
+@Configuration
+@ConditionalOnProperty(name = "grouping.api.server.type", havingValue = "OOTB")
+public class OotbSecurityConfig {
+
+    private static final Log logger = LogFactory.getLog(OotbSecurityConfig.class);
+    @Value("${grouping.api.server.type}")
+    String serverType;
+
+    @Value("${ootb.default.user.name}")
+    String name;
+
+    @Value("${ootb.default.user.username}")
+    String username;
+
+    @Value("${ootb.default.user.authorities}")
+    List<String> authorities;
+    @Value("${url.base}")
+    private String appUrlBase;
+
+    @Bean
+    public AuthenticationManager authenticationManager(AuthenticationConfiguration authenticationConfiguration)
+            throws Exception {
+        return authenticationConfiguration.getAuthenticationManager();
+    }
+
+    @Bean
+    public DelegatingAuthenticationFailureHandler authenticationFailureHandler() {
+        return new DelegatingAuthenticationFailureHandler(appUrlBase);
+    }
+
+    @Bean
+    public DelegatingWsConfiguration delegatingWsConfiguration() {
+        return new DelegatingWsConfiguration();
+    }
+
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+        http
+                .addFilterBefore(ootbStaticUserAuthenticationFilter(), UsernamePasswordAuthenticationFilter.class)
+                .authorizeHttpRequests((auths)
+                        -> auths.requestMatchers(antMatcher("/")).permitAll()
+                        .anyRequest().authenticated())
```

Comment:
Question, is there a reason that you chose all request matchers to be permit all?

Reply:
I wrote that code in the very beginning to make it easy to implement bypass authentication before I was able to set default user authorities. Since we can set the default user authorities through properties file now, I think I can make it all request matchers same as current existed security config. Thanks!

Reply:
Everything works well in the ootb ui after adding same requestMatchers as security config class.